### ISSUE TITLE
win_region issue with certain location codes

### ISSUE THREAD

**Issue by myllynen** [ansible-collections/ansible.windows#349]
##### SUMMARY
win_region fails to apply location code 39070 (World) and prints warning about junk after JSON data when using it.

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME
community.windows.win_region

##### ANSIBLE VERSION
```paste below
ansible [core 2.12.4]
  config file = None
  configured module search path = ['/home/testuser/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/testuser/.local/lib/python3.9/site-packages/ansible
  ansible collection location = /home/testuser/.ansible/collections:/usr/share/ansible/collections
  executable location = /home/testuser/.local/bin/ansible
  python version = 3.9.6 (default, Aug 19 2021, 00:00:00) [GCC 11.2.1 20210728 (Red Hat 11.2.1-2)]
  jinja version = 3.1.1
  libyaml = True
```

##### COLLECTION VERSION
```paste below
Collection        Version
----------------- -------
community.windows 1.9.0  
```

##### CONFIGURATION
```paste below

```

##### OS / ENVIRONMENT
Seen with ansible-core 2.11 + ansible-4.0 on RHEL 8 (Python 3.6) and with ansible-core-2.12 + ansible-5.0 on RHEL 9 (Python 3.9).

Target Windows system is Windows Server 2022 eval. Happens without patching or with all updates applied as of 2022-04-08.

##### STEPS TO REPRODUCE
```yaml
---
- name: Configure Windows region
  hosts: all
  tasks:
    - name: Configure system region
      win_region:
        location: 39070
        format: en-US
        unicode_language: en-US
        copy_settings: true
      register: region_config

    - name: Reboot system
      win_reboot:
        pre_reboot_delay: 0
        reboot_timeout: 600
      when: region_config.restart_required
```

##### EXPECTED RESULTS
Playbook changes the system only once and does not print warning about junk data.

##### ACTUAL RESULTS
Playbook reports changes after every run and prints warning about junk.

```paste below
 ansible-playbook -i 192.168.122.110, region.yml 

PLAY [Configure Windows region] ************************************************

TASK [Gathering Facts] *********************************************************
ok: [192.168.122.110]

TASK [Configure system region] *************************************************
[WARNING]: Module invocation had junk after the JSON data:  HKU
changed: [192.168.122.110]

TASK [Reboot system] ***********************************************************
skipping: [192.168.122.110]

PLAY RECAP *********************************************************************
192.168.122.110            : ok=2    changed=1    unreachable=0    failed=0    skipped=1    rescued=0    ignored=0   
```

It seems that if I use location codes 77 (Finland) and 244 (United Stated) then I still get the warning about junk when changing the value but after that the task does no changes. So with these two codes the issue is only that warning about junk when doing the change. But when using the location code 39070 (World) change reported always and also the warning about junk.

On Windows Settings view 77/244 are as expected after Ansible run. World is not available in the dropdown menu so looks like there's an inconsistency somewhere between known location codes and accepted location codes for the setting.

I will avoid using 39070 (World) for now so this is definitely not a critical issue. Thanks.

**Comment by myllynen** [comment]
I also see codes like 254 (Namibia) getting rejected, perhaps I'm looking at the wrong list of codes, however many of them do work. This was the list I was using as the reference:

https://docs.microsoft.com/windows/win32/intl/table-of-geographical-locations

**Comment by myllynen** [comment]
I see that I accidentally filed this issue at ansible.windows but this should be at community.windows where win_region lives. Could you please move this there if possible or if not I can create a new ticket there. Thanks!
